hasher-matcher-actioner/webapp/src/components/ThreatExchangePrivacyGroupCard.jsx

```diff
@@ -63,7 +66,15 @@ export default function ThreatExchangePrivacyGroupCard({
                   overlay={
                     <Popover id={`popover-basic${privacyGroupId}`}>
                       <Popover.Title as="h3">Information</Popover.Title>
-                      <Popover.Content>{description}</Popover.Content>
+                      <Popover.Content>
+                        <div
+                          dangerouslySetInnerHTML={{
+                            __html: DOMPurify.sanitize(`${description}`, {
+                              ADD_ATTR: ['target'],
+                            }),
+                          }}
```

Comment:
I don't know the in's and out's of SQL injections so going to defer to you and @schatten here if you think this is safe. I will note that these fields are editable by the privacy group owners, not just by FB.